### PR TITLE
Number port changes

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

**Comment by captainbeardo** [comment]
:+1:

File: lib/conduit/reactor/parsers/base.rb [hunk 1]
```diff
@@ -68,9 +68,9 @@ def unexpected_response_hash
       def object_path(path)
         data = json
 
-        path.to_s.split('/').map do |element|
+        value = path.to_s.split('/').map do |element|
```
Comment (arvanasse) [inline lib/conduit/reactor/parsers/base.rb]: Do we need to assign to a local variable here?  ```value``` does not seem to be used.

Comment (dougperez) [inline lib/conduit/reactor/parsers/base.rb]: We do not, fixed.

**Comment by arvanasse** [comment]
:+1:

**Comment by dougperez** [comment]
Merging this, I reverted the commit that remove errors. We can have a convo and figure out best approach.

**Pull request merged**